core/node/state_keeper/src/updates/mod.rs

```diff
@@ -35,6 +36,7 @@ pub struct UpdatesManager {
     base_fee_per_gas: u64,
     base_system_contract_hashes: BaseSystemContractsHashes,
     protocol_version: ProtocolVersionId,
+    pub storage_view_cache: StorageViewCache,
```

Comment:
nit: let's make `storage_view_cache` optional and set it to `Some` in a setter. Right now you can't distinguish if the setter was called and cache is empty indeed or it setter wasn't called